Bottom: 2d8f6970d59cbce4cb58f13b050373bf8328e1d8
Top:    567706d0f4370805507f88c61e9923f64a5f4738
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-24 23:57:57 +0100

Refresh of hid-gtk-don-t-compensate-for-c

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 8fd11ef..dc2ea56 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1837,11 +1837,7 @@ ScrollAction (int argc, char **argv, Coord x, Coord y)
   else
     AFAIL (scroll);
 
-  notify_crosshair_change (false);
   ghid_port_ranges_pan (dx, dy, TRUE);
-  MoveCrosshairRelative (dx, dy);
-  AdjustAttachedObjects ();
-  notify_crosshair_change (true);
 
   return 0;
 }
